tests/aws/aws_environment.rs

```diff
@@ -106,6 +106,45 @@ fn deploy_a_working_environment_with_no_router_on_aws_eks() {
     })
 }
 
+#[cfg(feature = "test-aws-self-hosted")]
+#[test]
+fn deploy_a_working_environment_and_pause_it_eks() {
+    engine_run_test(|| {
+        let test_name = "deploy_a_working_environment_and_pause_it";
+        let span = span!(
+            Level::INFO,
+            "test",
+            name = test_name
+        );
+        let _enter = span.enter();
+
+        let context = context();
+        let context_for_delete = context.clone_not_same_execution_id();
+        let secrets = FuncTestsSecrets::new();
+        let environment = test_utilities::aws::working_minimal_environment(&context, secrets);
+
+        let ea = EnvironmentAction::Environment(environment);
+        match deploy_environment(&context, &ea) {
+            TransactionResult::Ok => assert!(true),
+            TransactionResult::Rollback(_) => assert!(false),
+            TransactionResult::UnrecoverableError(_, _) => assert!(false),
+        };
+
+        match ctx_pause_environment(&context_for_delete, &ea) {
+            TransactionResult::Ok => assert!(true),
+            TransactionResult::Rollback(_) => assert!(false),
+            TransactionResult::UnrecoverableError(_, _) => assert!(false),
+        };
+
```

Comment:
don't you add any check to validate the result?

Reply:
it is the plan, when I get back the test cluster